### PR TITLE
Fixed relational attributes import issue

### PR DESCRIPTION
**Pull request by sajagana**
Note:

Fixed relational attributes import issue in the data source and resource files

### CODE REVIEW

File: aci/data_source_aci_infraaccnodepgrp.go [hunk 1]
```diff
@@ -14,14 +15,81 @@ func dataSourceAciAccessSwitchPolicyGroup() *schema.Resource {
 		ReadContext:   dataSourceAciAccessSwitchPolicyGroupRead,
 		SchemaVersion: 1,
 		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
```
Comment (akinross) [inline aci/data_source_aci_infraaccnodepgrp.go:17]: you can have a look at leveraging https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/aci/base_attr_schema.go#L42

File: aci/data_source_aci_infraaccbndlgrp.go [hunk 2]
```diff
@@ -56,5 +169,199 @@ func dataSourceAciPCVPCInterfacePolicyGroupRead(ctx context.Context, d *schema.R
 	if err != nil {
 		return diag.FromErr(err)
 	}
+
+	// infraRsSpanVSrcGrp - Beginning Read
+	log.Printf("[DEBUG] %s: infraRsSpanVSrcGrp - Beginning Read with parent DN", dn)
+	_, err = getAndSetReadRelationinfraRsSpanVSrcGrpFromPCVPCInterfacePolicyGroup(aciClient, dn, d)
+	if err != nil {
+		log.Printf("[DEBUG] %s: infraRsSpanVSrcGrp - Read finished successfully", d.Get("relation_infra_rs_span_v_src_grp"))
```
Comment (akinross) [inline aci/data_source_aci_infraaccbndlgrp.go:177]: Read finished successfully, is this correct?

File: aci/data_source_aci_infraaccnodepgrp.go (hunk 1 above)
Comment (sajagana) [reply aci/data_source_aci_infraaccnodepgrp.go:17]: Helpful, Thanks @akinross

**Review by lhercot — CHANGES_REQUESTED** [review]
Please also do a rebase on latest on top of below comments

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**